### PR TITLE
Added background and nav

### PR DESCRIPTION
**Pull request by suraj385**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,32 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Bootstrap Example</title>
+  <title>Soch traning</title>
```
Comment (AcharyaShreedhar) [inline index.html]: correct spelling of training

File: index.html [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
 </head>
 <body>
-
+<body background="download.jpg">
 <nav class="navbar navbar-default">
   <div class="container-fluid">
     <div class="navbar-header">
-      <a class="navbar-brand" href="#">Workshop Information System</a>
+      <a class="navbar-brand" href="#">Soch College of IT</a>
```
Comment (AcharyaShreedhar) [inline index.html:16]: you are not supposed to change this

**Review by AcharyaShreedhar — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (suraj385) [reply index.html]: done

File: index.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
 <div class="container">
   <h3> Training</h3>
   <p>
-" This is basic page"</p>
+" This is basic page"</p>add
```
Comment (AcharyaShreedhar) [inline index.html]: add ?

**Review by AcharyaShreedhar — CHANGES_REQUESTED** [review]

**Comment by AcharyaShreedhar** [comment]
Also can you please resolve conflicts

**Comment by AcharyaShreedhar** [comment]
Can you change the base branch?

File: index.html (hunk 3 above)
Comment (suraj385) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (suraj385) [reply index.html:16]: Done

**Review by AcharyaShreedhar — APPROVED** [review]

**Pull request merged**